src/course-team/utils.js

```diff
@@ -0,0 +1,44 @@
+import { MODAL_TYPES } from './enum';
+import messages from './info-modal/messages';
+
+const getErrorEmailFromMessage = (message) => {
+  const regex = /'([^']+)'/g;
+  return message.match(regex)[0];
+};
+
+const getInfoModalSettings = (modalType, currentEmail, errorEmail, courseName, intl) => {
+  switch (modalType) {
+  case MODAL_TYPES.delete:
```

Comment:
Up to you

`const { delete, error, warning } = MODAL_TYPES;`

Reply:
delete - is reserved word